OIPA/iati_organisation/parser/organisation_2_03.py

```diff
@@ -942,6 +942,147 @@ def iati_organisations__iati_organisation__recipient_region_budget__budget_line_
         self.add_narrative(element, recipient_region_budget_line)
         return element
 
+    def iati_organisations__iati_organisation__recipient_country_budget(
+            self, element):
+        status = element.attrib.get("status")
+        if status:
+            status = self.get_or_none(codelist_models.BudgetStatus,
+                                      code=status)
+            if status is None:
+                raise FieldValidationError(
+                    "RecipientCountryBudget",
+                    "status",
+                    "not found on the accompanying codelist.",
+                    None,
+                    None,
+                )
+        else:
+            status = codelist_models.BudgetStatus.objects.get(code=1)
+        recipient_country = element.xpath("recipient-country")
```

Comment:
I don't think think that squasing child elements in one single parser method with `xpath` (in this case - `recipient-country`) is a good idea.